src/main/java/de/unistuttgart/towercrushbackend/controller/websockets/WebsocketController.java

```diff
@@ -1,44 +1,54 @@
 package de.unistuttgart.towercrushbackend.controller.websockets;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import de.unistuttgart.towercrushbackend.data.websockets.JoinTeamMessage;
-import de.unistuttgart.towercrushbackend.data.websockets.MessageWrapper;
-import de.unistuttgart.towercrushbackend.data.websockets.Purpose;
+import de.unistuttgart.towercrushbackend.data.websockets.*;
+import de.unistuttgart.towercrushbackend.service.websockets.LobbyManagerService;
+import de.unistuttgart.towercrushbackend.service.websockets.WebsocketService;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.Header;
 import org.springframework.messaging.handler.annotation.MessageMapping;
 import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Controller;
 
+import java.security.Principal;
+
 
 @Controller
 @Slf4j
 public class WebsocketController {
 
     @Autowired
     SimpMessagingTemplate simpMessagingTemplate;
-    final String lobbyDestination = "/topic/lobbies/";
+    @Autowired
+    LobbyManagerService lobbyManagerService;
+    @Autowired
+    WebsocketService websocketService;
+    final String newPlayerQueue = "/queue/private/messages";
+    final String lobbyTopic = "/topic/lobby/";
```

Comment:
```suggestion
    static final String NEW_PLAYER_QUEUE = "/queue/private/messages";
    static final String LOBBY_TOPIC = "/topic/lobby/";
```
Naming convention for constants

Reply:
idk why but sure